src/Bridge/Symfony/Messages/BatchOfUpdatesMessage.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Search\Bridge\Symfony\Messages;
+
+use LoyaltyCorp\Search\Bridge\Symfony\Interfaces\SearchMessageInterface;
+
+final class BatchOfUpdatesMessage implements SearchMessageInterface
```

Comment:
Why cant you use the BatchOfUpdatesEvent? It isnt specific to laravel/lumen/symfony